share/logcfg.dat

```diff
@@ -68,8 +68,8 @@ TIME_MASTER
 #################################
 #  use 'addnode' only for OTHER nodes !!
 #
-#ADDNODE=10.0.0.115
-#ADDNODE=192.168.1.2
+#NODE_B=10.0.0.115
+#NODE_C=192.168.1.2
```

Comment:
Do we need a NODE_A entry for our-self?

Reply:
My concerns were about the display of network data for our own node in `networkinfo`. As far as I see not having a node for our-self leads to the according bc_hostaddress not set and that leads to `networkinfo`  not showing data for our node. 
(Not sure if I am correct here, just glancing over the code).

Reply:
Did some more reading. Network data for our own node stays always 0. So no real need to show them. Feel free to ignore my comment above.

Reply:
Yes, we show the sent packets per node, so our will always be zero.
Normally all others shall be equal unless there were errors, as we broadcast to all nodes.
Regarding the errors: in a typical setup (i.e. hosts could be looked up) I see no real possibility of UDP sending failing to a particular node. Sending works even if the node is down. So maybe only if the network/interface down. Anyway, these stats doesn't harm even if they are not too useful.